.gitignore

```diff
@@ -61,3 +61,10 @@
 .idea
 .vscode/ipch
 /newIDE/app/src/UI/Theme/**/*ThemeVariables.*
+.DS_Store
+.DS_Store?
```

Comment:
Is this one really necessary? Let's keep just the first one.

Reply:
Okay